### PR TITLE
implement of transaction payload of Record.

### PR DESCRIPTION
**Pull request by luodanwg**
add the feature of record transaction. Add grpc test func.

Signed-off-by: junjie luodan.wg@gmail.com

### CODE REVIEW

File: core/store/LevelDBStore/LevelDBStore.go [hunk 1]
```diff
@@ -468,7 +468,8 @@ func (bd *LevelDBStore) persist(b *Block) error {
 	for i := 0; i < nLen; i++ {
 
 		// now support RegisterAsset / IssueAsset / TransferAsset and Miner TX ONLY.
-		if b.Transactions[i].TxType == tx.RegisterAsset || b.Transactions[i].TxType == tx.IssueAsset || b.Transactions[i].TxType == tx.TransferAsset || b.Transactions[i].TxType == tx.BookKeeping {
+		if b.Transactions[i].TxType == tx.RegisterAsset || b.Transactions[i].TxType == tx.IssueAsset || b.Transactions[i].TxType == tx.TransferAsset ||
+			b.Transactions[i].TxType == tx.Record || b.Transactions[i].TxType == tx.BookKeeping {
```
Comment (dreamfly281) [inline core/store/LevelDBStore/LevelDBStore.go]: each line's length no more than 80 characters is a good style

File: core/transaction/TransactionBuilder.go [hunk 2]
```diff
@@ -63,4 +63,23 @@ func NewTransferAssetTransaction() (*Transaction, error){
         Payload: assetRegPayload,
         Programs: []*program.Program{},
     }, nil
+}
+
+//initial a new transaction with record payload
+func NewRecordTransaction(recordType string, recordData []byte) (*Transaction, error) {
+
```
Comment (dreamfly281) [inline core/transaction/TransactionBuilder.go]: don't need blank line here

File: core/store/LevelDBStore/LevelDBStore.go (hunk 1 above)
Comment (luodanwg) [inline core/store/LevelDBStore/LevelDBStore.go]: I have refomate it , could you helps check it again, thanks.

File: core/transaction/TransactionBuilder.go (hunk 2 above)
Comment (luodanwg) [inline core/transaction/TransactionBuilder.go]: thanks, i have delete this blank row.

**Pull request merged**